source/class/qx/tool/cli/commands/MConfig.js

```diff
@@ -251,6 +252,20 @@ qx.Mixin.define("qx.tool.cli.commands.MConfig", {
       if (argv.locale && argv.locale.length) {
         result.locales = argv.locale;
       }
+
+      if (argv["set-env"]) {
+        argv["set-env"].forEach(function(kv) {
+          var m = kv.match(/^([^=\s]+)(=(.+))?$/);
```

Comment:
The regex will not recognize `--set-env foo = bar` which might be sensible but this should be documented. It accepts `--set-env foo`, setting foo to be undefined - is there any use in this? Maybe setting a value of a key should be enforced?

Reply:
It works without spaces around the equal sign. i.e. `--set-env foo=bar`. It is the same regexp used for the `--set` option. Otherwise the shell or yargs would probably get confused and consider `foo`, `=` and `bar` as 3 different parameters.